docs/design/operator-bundle.md

```diff
@@ -151,24 +159,29 @@ Flags:
 The `--directory/-d`, `--channels/-c`, `--package/-p` are required flags while `--default/-e` and `--output-dir/-u` are optional.
 
 The command for `generate` task is:
+
 ```bash
 $ ./opm alpha bundle generate --directory /test --package test-operator \
 --channels stable,beta --default stable
 ```
 
 The `--directory` or `-d` specifies the directory where the operator manifests, including CSVs and CRDs, are located. For example:
+
 ```bash
 $ tree test
 test
 ├── etcdcluster.crd.yaml
 └── etcdoperator.clusterserviceversion.yaml
 ```
 
-The `--package` or `-p` is the name of package fo the operator such as `etcd` which which map `channels` to a particular application definition. `channels` allow package authors to write different upgrade paths for different users (e.g. `beta` vs. `stable`). The `channels` list is provided via `--channels` or `-c` flag. Multiple `channels` are separated by a comma (`,`). The default channel is provided optionally via `--default` or `-e` flag. If the default channel is not provided, the first channel in channel list is selected as default.
+The `--package` or `-p` is the name of package of the operator such as `etcd`, which maps `channels` to a particular application definition. `channels` allow package authors to write different upgrade paths for different users (e.g. `beta` vs. `stable`). The `channels` list is provided via the `--channels` or `-c` flag. Multiple `channels` are separated by a comma (`,`). The default channel is provided optionally via `--default` or `-e` flag. If the default channel is not provided, the first channel in the list of channels is selected as the default.
+
+**Note**: All information specified in `annotations.yaml` must also exist in the `LABEL` section of a `Dockerfile`.
+
+After the generate command is executed, the `Dockerfile` is generated in the same directory where the command was run. By default, the `annotations.yaml` file is located in a folder named `metadata` in the same root directory as the input directory containing manifests.
```

Comment:
This whole section on `opm alpha bundle` commands should probably be either revised or removed based on the outcome of https://github.com/operator-framework/operator-registry/pull/563

Reply:
I'm fine with removing this prematurely to that PR landing if the intention is we're moving all of that bundle generation functionality over to operator-sdk.

Reply:
Done - added a separate commit that removes that section entirely.